core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPathfinding.kt

```diff
@@ -265,7 +233,10 @@ class STDCMPathfinding(
                         .setStartOffset(location.offset)
                         .setPrevMaximumAddedDelay(maxDepartureDelay)
                         .makeAllEdges()
-                for (edge in edges) res.add(edge)
+                for (edge in edges) {
+                    if (edge.previousNode != null) res.add(edge.previousNode)
```

Comment:
I don't think this can happen

Reply:
This is null if it's the first edge, so this takes into account and adds the path's very first node.

Reply:
But the edges have just been created and they don't have any previous node